src/osal.h

```diff
@@ -117,6 +121,14 @@ enum os_eth_type {
 typedef uint32_t os_ipaddr_t;
 typedef uint16_t os_ipport_t;
 
+/**
```

Comment:
pnet_ethaddr_t should be remove if os_ethaddr_t is added. These types describe the same thing.

Reply:
Done

Reply:
Sorry, didn't think this through. With this change pnet_api.h now depends on osal.h, and the installed build doesn't work unless the client also adds osal.h to the include search path. The point of adding pnet_ethaddr_t was to avoid depending on osal.h in the first place.

Reply:
Did put back pnet_ethaddr_t